infrastructure/local/docker-compose.yml

```diff
@@ -21,14 +22,13 @@ services:
       - redis
       - vault
     environment:
-      # This environment vars overwrite the ones in config.toml
-      - SH_ID_PLATFORM_DATABASE_URL=postgres://polygonid:polygonid@postgres:5432/platformid?sslmode=disable
-      - SH_ID_PLATFORM_KEY_STORE_ADDRESS=http://vault:8200
       - SH_ID_PLATFORM_KEY_STORE_TOKEN=${KEY_STORE_TOKEN}
-      - SH_ID_PLATFORM_REDIS_URL=redis://@redis:6379/1
+    env_file:
+      - ../../.shared.env
+      - ../../.env
```

Comment:
Is this the best structure of the env files? In my understanding, we would want only the variables per container that are needed. There should probably be an env file for every container with its unique variables, so that we are not reading e.g. `ETHEREUM` vars into the api-ui.

Reply:
That is the intention. We should use:
* `.env-issuer` for the API issuer node and API UI. (both services shared a lot of envs)
* `.env-shared`. for shared variables between UI and UI backend.
* `.ui.env` for UI service.

Reply:
Okay so every var for the issuer API is also used in the UI API? If not, they should probably be distinct.

Reply:
we are not 100% sure yet, but we think at this moment that both are going to use almost the same. If we are wrong, we can create anew one.